src/signature/mod.rs

```diff
@@ -0,0 +1,12 @@
+pub mod basic_lamport;
```

Comment:
Didn't know about `mod.rs` files! Looks like it's now [recommended](https://doc.rust-lang.org/reference/items/modules.html#module-source-filenames) to move this file to `src/signature.rs`, so I think I'll do that.

Reply:
Sounds good!